iset.mm

```diff
@@ -118099,6 +118104,495 @@ Infinite sums (cont.)
+    mulcl oveq12d halfaddsubcl 2timesd cossub cosadd coscl pnncand eqtrd simprd
+    halfaddsub fveq2d simpld 3eqtr2rd ) ACDBCDEZFABGHFIHZJKZABLHFIHZJKZMHZMHURU
+    RGHZUNUPLHZNKZUNUPGHZNKZLHZBNKZANKZLHUMURUMUNCDZUPCDZEZURCDZABUAZVGUOCDUQCD
+    VJVHUNOUPOUOUQSPQZUBUMVDUNNKZUPNKZMHZURGHZVOURLHZLHZUSUMVIVDVRRVKVIVAVPVCVQ
+    LUNUPUCUNUPUDTQUMVOURURUMVIVOCDZVKVGVMCDVNCDVSVHUNUEUPUEVMVNSPQVLVLUFUGUMVA
+    VEVCVFLUMUTBNUMVBARZUTBRZABUIZUHUJUMVBANUMVTWAWBUKUJTUL $.
+
+  $( Sine squared plus cosine squared is 1.  Equation 17 of [Gleason] p. 311.
+     Note that this holds for non-real arguments, even though individually each
+     term is unbounded.  (Contributed by NM, 15-Jan-2006.) $)
+  sincossq $p |- ( A e. CC ->
+                  ( ( ( sin ` A ) ^ 2 ) + ( ( cos ` A ) ^ 2 ) ) = 1 ) $=
+    ( cc wcel cneg caddc co ccos cfv cmul csin c1 cexp cc0 sqvald oveq2d eqtr4d
+    c2 sqcld eqtrd mulcld cmin wceq negcl cosadd mpdan negid fveq2d cos0 syl6eq
+    addcomd cosneg sinneg negeqd negnegd sincld mulneg2d oveq12d coscld negsubd
+    sincl coscl 3eqtrrd 3eqtr3rd ) ABCZAADZEFZGHZAGHZVEGHZIFZAJHZVEJHZIFZUAFZKV
+    KQLFZVHQLFZEFZVDVEBCVGVNUBAUCZAVEUDUEVDVGMGHKVDVFMGAUFUGUHUIVDVQVPVOEFVJVMD
+    ZEFVNVDVOVPVDVKAUTZRVDVHAVAZRUJVDVPVJVOVSEVDVPVHVHIFVJVDVHWANVDVIVHVHIAUKOP
+    VDVOVKVLDZIFZVSVDVOVKVKIFWCVDVKVTNVDWBVKVKIVDWBVKDZDVKVDVLWDAULUMVDVKVTUNSO
+    PVDVKVLVTVDVEVRUOZUPSUQVDVJVMVDVHVIWAVDVEVRURTVDVKVLVTWETUSVBVC $.
+
+  $( Double-angle formula for sine.  (Contributed by Paul Chapman,
+     17-Jan-2008.) $)
+  sin2t $p |- ( A e. CC ->
+      ( sin ` ( 2 x. A ) ) = ( 2 x. ( ( sin ` A ) x. ( cos ` A ) ) ) ) $=
+    ( cc wcel c2 cmul co csin cfv caddc ccos 2times fveq2d coscl mulcomd oveq2d
+    sincl wceq sinadd anidms mulcld 2timesd 3eqtr4d eqtrd ) ABCZDAEFZGHAAIFZGHZ
+    DAGHZAJHZEFZEFZUDUEUFGAKLUDUJUIUHEFZIFZUJUJIFUGUKUDULUJUJIUDUIUHAMZAPZNOUDU
+    GUMQAARSUDUJUDUHUIUOUNTUAUBUC $.
+
+  $( Double-angle formula for cosine.  (Contributed by Paul Chapman,
+     24-Jan-2008.) $)
+  cos2t $p |- ( A e. CC -> ( cos ` ( 2 x. A ) ) =
+                           ( ( 2 x. ( ( cos ` A ) ^ 2 ) ) - 1 ) ) $=
+    ( cc wcel ccos cfv c2 cexp co c1 cmin caddc cmul coscl sqcld ax-1cn subsub3
+    wceq syl2anc sqvald 3eqtr4d mp3an2 csin cosadd anidms 2times fveq2d oveq12d
+    sincl addcomd sincossq eqtr3d wb subadd mp3an1 mpbird oveq2d eqtr4d 2timesd
+    oveq1d ) ABCZADEZFGHZIVBJHZJHZVBVBKHZIJHZFALHZDEZFVBLHZIJHUTVBBCZVJVDVFQZUT
+    VAAMZNZVMVJIBCZVJVKOVBIVBPUARUTVHVBAUBEZFGHZJHZVDUTAAKHZDEZVAVALHZVOVOLHZJH
+    ZVHVQUTVSWBQAAUCUDUTVGVRDAUEUFUTVBVTVPWAJUTVAVLSUTVOAUHZSUGTUTVCVPVBJUTVCVP
+    QZVBVPKHZIQZUTVPVBKHWEIUTVPVBUTVOWCNZVMUIAUJUKUTVJVPBCZWDWFULZVMWGVNVJWHWIO
+    IVBVPUMUNRUOUPUQUTVIVEIJUTVBVMURUST $.
+
+  $( Double-angle formula for cosine in terms of sine.  (Contributed by NM,
+     12-Sep-2008.) $)
+  cos2tsin $p |- ( A e. CC -> ( cos ` ( 2 x. A ) ) =
+                           ( 1 - ( 2 x. ( ( sin ` A ) ^ 2 ) ) ) ) $=
```

Comment:
IMO cos(2A) = (cos A)^2 -  (sin A)^2 is worth having in (i)set.mm and it could probably be used to prove both cos2t and cos2tsin and globally shorten the proofs. Just browsing nearby theorems on sums and products of sines and cosines, there might be some other shortening opportunities. Any takers ? @avekens maybe ?